src/main/kotlin/com/exactpro/th2/codec/configuration/Configuration.kt

```diff
@@ -49,43 +41,25 @@ class Configuration() {
     var generalEncoderInputAttribute: String = "general_encoder_in"
     var generalEncoderOutputAttribute: String = "general_encoder_out"
```

Comment:
If we are changing that part anyway let's make those attributes constant (all 8 fields). It must not be allowed to change them.